users.md

```diff
@@ -1,6 +1,476 @@
 ---
 layout: contribute
-title: Bazel Users
+title: Who is using Bazel
 ---
 
-Find a community curated a list of Bazel users in the [Bazel GitHub wiki](https://github.com/bazelbuild/bazel/wiki/Bazel-Users)
\ No newline at end of file
+*Using Bazel? Send an email to [product@bazel.build](mailto:produc@bazel.build) to get added to the list.
```

Comment:
*Using Bazel? You can add your company on [StackShare](https://stackshare.io/bazel). To add yourself to this page, contact [product@bazel.build](mailto:produc@bazel.build).*

(Flipping the order of things they can do without assistance first and ending with the call to action.)

Reply:
That works for me.